src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -4388,6 +4384,163 @@ static bool M(Func<int> func)
             var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
 
+        [WorkItem(48563, "https://github.com/dotnet/roslyn/issues/48563")]
+        [Theory]
+        [InlineData("void*")]
+        [InlineData("char*")]
+        [InlineData("delegate*<void>")]
```

Comment:
Consider also using `T*` when `T` is a generic parameter that has the unmanaged constraint

Reply:
It looks like we don't allow `is null` and `is not null` for those cases: see [https://github.com/dotnet/roslyn/issues/48591](https://github.com/dotnet/roslyn/issues/48591)

---
In reply to: [504771996](https://github.com/dotnet/roslyn/pull/48585#discussion_r504771996) [](ancestors = 504771996)

Reply:
Interesting. That definitely seems like an ommission on our part. Thanks for filing the tracking issue.